.travis.yml

```diff
@@ -18,16 +18,15 @@ matrix:
       python: pypy
     - os: linux
       python: pypy3
-# Perform the manual steps on osx to install python3 and activate venv
 before_install:
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update          ; fi
+  # Perform the manual steps on osx to install python3 and activate venv
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install python3; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then virtualenv venv -p python3; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then source venv/bin/activate; fi
```

Comment:
I cannot guess the syntax of this file from this: we have separate ifs as if each line worked in its own shell, then source a script for use at other lines.